.changeset/real-masks-roll.md

```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/console": minor
```

Comment:
Shall we bump i18n and myaccount as well?

Reply:
Addressed fd29c6070019dd2193690e6d85fcae670bd64815